### DISCUSSION TITLE
Discusión General

### DISCUSSION THREAD

**Discussion by mora145** [Ludeon/RimWorld-SpanishLatin#77] [General]
Saludos @zerstrick. 
Al respecto de las ultimas actualizaciones y el DLC... ¿Cómo estas haciendo para agregar las nuevas lineas de texto y los cambios a las etiquetas, te estas apoyando en el RimTrans?

**Comment by zerstrick** [comment]
Saludos @mora145  el RimTrans ya no se usa, únicamente para los mods, todo el texto del DLC ya lo tengo y empecé a traducirlo, el juego incluye una herramienta interna que limpia la traducción, genera los textos nuevos y sus respectivas etiquetas de forma automática.

**Reply by mora145** [reply]
Me temía que ya no se usara.
¿Podrías subir lo que llevas para ir revisando al menos?

**Reply by mora145** [reply]
¿Ya aprendiste a usar el GitHub Desktop?

**Reply by zerstrick** [reply]
Empecé con un archivo muy grande aún no lo termino iré subiendo a medida que vaya terminando los archivos, así como estaba haciendo con el parche 1.3

**Reply by zerstrick** [reply]
Empecé con un archivo muy grande aún no lo termino iré subiendo a medida que vaya terminando los archivos, así como estaba haciendo con el parche 1.3

**Reply by mora145** [reply]
¿Qué mecanismo están usando para llevar una traducción mancomunada? ¿En que archivo te estas centrando?

**Reply by zerstrick** [reply]
Sí, el GitHub Desktop es mucho más práctico que estar usando comandos en el VSC, llevo bastante ya usándolo, hasta para mi repositorio donde tengo todos los mods con los que he ido trabajando.

**Reply by zerstrick** [reply]
No entiendo la pregunta de la traducción mancomunada, hay 3 futuros colaboradores que están esperando acceso al github para ayudar, mientras yo empiezo a trabajar con los keyed, luego cuando se coordinen conmigo quienes deseen ayudar a traducir este DLC empiezo a repartir tareas para que todos estemos haciendo algo diferente, por ahora estoy estudiando los nombres nuevos mientras trato de adecuarme a todo para luego se mantenga ese nombre para todo. No sé si eso responde la pregunta. A la par también estoy colaborando con la versión castellana.

**Reply by mora145** [reply]
Exacto, me refiero a eso. Si se está traduciendo en grupo o no, y por medio de que método se están organizando. 
Te lo digo, porque antes subíamos los textos aun sin traducirse y se asignaban traductores a cada archivo, para evitar conflictos de traducción. De todas maneras me gustaría ayudar con las revisiones que hagan falta, ya que no he tenido tiempo para entrarle full a la traducción como tal.

**Reply by zerstrick** [reply]
Sí, la idea es subir los archivos >TODO< asignados, esto para evitar que los nuevos quieran saltar de uno a otro, mi idea es dar carpetas para que trabajen, pero por ahora necesito que tengan acceso a github, como dije iré subiendo a medida que avance para llevar un mejor control porque con la nueva herramienta propia del juego cuando hay cambios de textos en los comentarios de una traducción previa si no estas pendiente suele pasar que no se hace el cambio porque no es una archivo TODO y se salta ya que se considera traducido evitando añadir o eliminar la información nueva, por eso no me gusta subir una actualización pasando la herramienta del juego de primero porque esos archivos modificados no quedan marcados en VSC.

**Reply by zerstrick** [reply]
Con respecto a la coordinación usamos mi servidor de Discord.

**Reply by mora145** [reply]
Ya entiendo. Entré en el Discord con apodo "Mora"; quedo a la disposición para tareas de revisión. Mientras, le daré una revisada general a la traducción buscando errores y demás. Saludos.

**Reply by zerstrick** [reply]
Genial, eso siempre se agradece porque con la 1.3 fue mucho trabajo para una sola persona en tan corto tiempo y no me dio tiempo hacer segunda revisión, aunque Andrés siempre está por ahí revisando otros ojos no caen mal.

**Reply by mora145** [reply]
En principio esperaba traducir, pero en vista de mi falta de tiempo y dedicación a otros proyectos, por lo menos apoyo con esto.
